src/app/components/device-list/deviceList.js

```diff
@@ -0,0 +1,54 @@
+import {Component} from 'angular2/core';
+
+@Component({
+    selector: 'device-list',
+    template: require('./deviceList.html')
+})
+export class DeviceList {
+    deviceList = [{
```

Comment:
lets fetch the device list from a server (mock the response with some tool, till #1 is merged)